lib/msal-node/src/client/ManagedIdentitySources/AzureArc.ts

```diff
@@ -64,13 +82,38 @@ export class AzureArc extends BaseManagedIdentitySource {
     }
 
     public static getEnvironmentVariables(): Array<string | undefined> {
```

Comment:
the name of this method no longer matches its behavior

Reply:
Well... I think it does. If real environment variables don't exist, it returns pre-determined ones (if the himds file exists). The method is still returning the same thing.